src/client/activation/migrateDataScienceSettingsService.ts

```diff
@@ -38,7 +38,7 @@ export class MigrateDataScienceSettingsService implements IExtensionActivationSe
             'MigratedDataScienceSettingsService',
             false
         );
-        if (!migrated.value) {
+        if (true) {
```

Comment:
Did you intend for this to still be 'true'?

Reply:
Dammit, no